### DISCUSSION TITLE
add code editing in github android apps

### DISCUSSION THREAD

**Discussion by FollowNaim** [community/community#5641] [Mobile] [answered]
please add editing system in github apps

**Comment by aconsuegra** [comment]
Thanks for your feedback! We are tracking this request internally and are looking into how we can bring this to the app.

Also, we have similar discussions with the same feature request. Feel free to upvote some of those!
https://github.com/github/feedback/discussions/5443
https://github.com/github/feedback/discussions/3164
https://github.com/github/feedback/discussions/3820

**Reply by FollowNaim** [reply]
You have been saying the same thing for six or seven months.
Do you really do it or not?
if yes then How long will it take?

**Reply by aconsuegra** [reply]
We are evaluating all possible scenarios, so it is impossible to say right now when code editing will land in the apps.

**Comment by spaltrowitz** [comment, marked as answer]
thanks for your suggestion @TheNaimBro! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code
